Sources/Sentry/Transactions/ISentrySpan.h

```diff
@@ -0,0 +1,17 @@
+//
+//  ISpan.h
+//  Sentry
+//
+//  Created by Dhiogo Brustolin on 05/01/21.
+//  Copyright © 2021 Sentry. All rights reserved.
+//
+
```

Comment:
```suggestion
```

Reply:
this could be configured on Xcode, so it does not add any more.